### ISSUE TITLE
stopApplicationAndUploadData() Crashes Android Devices

### ISSUE THREAD

**Issue by manifestinteractive** [uxcam/cordova-uxcam#6]
#### Overview:

Following the documentation to stop UXCam in Cordova works perfectly fine for iOS devices, however, on two different Android Devices ( both Nexus 5's ) the command caused the app to completely crash.  The session also fails to upload to uxcam.com until the app is relaunched.  Session is then uploaded and the crash is logged in your system.
#### Acceptance Criteria:

`UXCam.stopApplicationAndUploadData()` should not cause the application to crash on Android Devices.  User testing should correctly stop and data should be uploaded to uxcam.com
#### Steps to Duplicate:

I have tried both of the the following examples on two different Android devices getting the same response for each attempt to stop UXCam.

``` js
UXCam.stopApplicationAndUploadData(function () { console.log('stop success') }, function () { console.log('stop error') })
```

``` js
UXCam.stopApplicationAndUploadData()
```

I tried removing the callbacks in `stopApplicationAndUploadData` as I thought they might be causing the crash ( as callbacks are failing too, as reported in Issue #5 ).  But that did not resolve the issue.  Both ways of triggering the stop fails and immediately crashes the application upon execution.
#### System Info:

Android OS v6.0.1 ( Tested on two different Nexus 5's with failure )
#### Relevant Documentation

I am referencing the `stopApplicationAndUploadData` UXCam method in this file:

https://github.com/uxcam/cordova-uxcam/blob/master/www/UXCam.js
#### My feelings on this Issue

![throw_a_kid](https://cloud.githubusercontent.com/assets/508411/18226513/b58f625c-71d9-11e6-8cda-4851baa71832.gif)

**Comment by uxcam-android** [comment]
I have pushed fix on branch v2.3.3. Could you please verify? @manifestinteractive 
https://github.com/uxcam/cordova-uxcam/tree/v2.3.3

**Comment by manifestinteractive** [comment]
Finally got this into an Android specific build and I can verify that v2.3.3 resolves both Android issues I was having.  Installed plugin via: ( for others that might want to use this branch )

```
cordova plugin add https://github.com/uxcam/cordova-uxcam.git#v2.3.3
```
